x-pack/plugins/security_solution/public/management/hooks/response_actions/use_send_scan_request.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IHttpFetchError } from '@kbn/core-http-browser';
+import {
+  useMutation,
+  type UseMutationOptions,
+  type UseMutationResult,
+} from '@tanstack/react-query';
+import type { ResponseActionApiResponse } from '../../../../common/endpoint/types';
+import type { ScanActionRequestBody } from '../../../../common/api/endpoint';
+import { KibanaServices } from '../../../common/lib/kibana';
+import { SCAN_ROUTE } from '../../../../common/endpoint/constants';
+
+/**
+ * Create scan request
+ * @param customOptions
+ */
+export const useSendScanRequest = (
```

Comment:
can you add a tests for this hook. The test should validate that this hook correctly populates the request's `body`